x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/resilient/translations.ts

```diff
@@ -134,16 +134,16 @@ export const MAPPING_FIELD_COMMENTS = i18n.translate(
 );
 
 export const DESCRIPTION_REQUIRED = i18n.translate(
-  'xpack.triggersActionsUI.components.builtinActionTypes.common.requiredDescriptionTextField',
+  'xpack.triggersActionsUI.components.builtinActionTypes.resilient.requiredDescriptionTextField',
   {
     defaultMessage: 'Description is required.',
   }
 );
 
 export const TITLE_REQUIRED = i18n.translate(
-  'xpack.triggersActionsUI.components.builtinActionTypes.common.requiredTitleTextField',
+  'xpack.triggersActionsUI.components.builtinActionTypes.resilient.requiredTitleTextField',
```

Comment:
Should this reflect the change below? `defaultMessage: 'Name is required.',`

```suggestion
  'xpack.triggersActionsUI.components.builtinActionTypes.resilient.requiredNameTextField',
```